central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -208,9 +211,20 @@ func New(db *postgres.DB, clusterID string) FlowStore {
 		return nil
 	}
 
+	partitionName := fmt.Sprintf("network_flows_v2_%s", strings.ReplaceAll(clusterID, "-", "_"))
```

Comment:
This may be unnecessary, but I'm wary of having %s in the queries. Can we validate that cluster id is a uuid and ensure that someone couldn't create a sql injection in a cluster id?

Reply:
verifying that is a UUID first is a good idea.  the other option is to just generate a new id for the partition, but using the cluster seemed to make sense.

Reply:
👍